src/test/groovy/pl/futurecollars/invoicing/AppTest.groovy

```diff
@@ -7,7 +7,7 @@ import spock.lang.Specification
 
 class AppTest extends Specification {
 
-    def "simple test to cover main"() {
+    def "dummy test to cover main"() {
```

Comment:
Klasa App nie zawiera żadnej logiki. Może warto się jej pozbyć? Jeśli uznasz, że tak, to ten test również będzie do usunięcia.